presto-product-tests/conf/docker/acid/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+
+services:
+  hadoop-master:
+    image: 'prestodev/hdp3.1-hive'
+    hostname: hadoop-master
+    ports:
+      - '${HIVE_PROXY_PORT}:1180'
+      - '5006:5006'
+      - '8020:8020'
+      - '8042:8042'
+      - '8088:8088'
+      - '9000:9000'
+      - '9083:9083'
+      - '10000:10000'
+      - '19888:19888'
+      - '50070:50070'
+      - '50075:50075'
+      - '9866:9866'
+      - '9867:9867'
+      - '9870:9870'
+      - '9864:9864'
+    volumes:
```

Comment:
This extends the `hadoop-master` service definition from `standard.yaml`, so you should not need `ports`, `volumes` here.